rfcs/text/0027-faas-fields.md

```diff
@@ -122,6 +169,7 @@ e.g.:
 ## References
 
 <!-- Insert any links appropriate to this RFC in this section. -->
+* [OpenTelemetry Faas Specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/faas.md#example)
 
 ### RFC Pull Requests
```

Comment:
Can you link this PR in the below list for stage 1?

Reply:
done!